sources/PseudoTerminal.m

```diff
@@ -5596,6 +5603,7 @@ - (void)setDimmingForSession:(PTYSession *)aSession
         // broadcasting to the current session.
         [[aSession view] setDimmed:YES];
     }
+    [[aSession tab] recheckBlur];
```

Comment:
This method is only meant to adjust dimming. I see that it gets called from several places, so to avoid having recheckBlur calls littered all over the place, the simplest thing might be to rename it from `setDimmingForSession:` to `updateDimAndBlurForSession:`, and rename callers as needed (e.g., `setDimmingForSessions`). XCode's "refactoring" feature makes this pretty easy to do.